ios/Podfile.lock

```diff
@@ -652,31 +652,31 @@ PODS:
   - Yoga (1.14.0)
   - YogaKit (1.18.1):
     - Yoga (~> 1.14)
-  - ZendeskAnswerBotProvidersSDK (5.0.0):
```

Comment:
are those changes intended? We are bumping some of the libraries by a major and this may cause unwanted bugs

Reply:
@CrisTofani That's strange, because I just updated the `react-native-screens` package. I'll try running `pod install` again to see if those updates are removed.

Reply:
@CrisTofani I just did it again, but the result is the same. The `react-native-screens` update probably triggers a new major version for these sub-packages. But I wouldn't assume that this update alone could cause some bugs, it could fix them in the same way.

Reply:
I actually tried moving back all the versions locked in the .lock file except screens and anything looks fine

Reply:
I actually pushed the revert on `Podfile.lock` here [5a4b5fb](https://github.com/pagopa/io-app/pull/6340/commits/5a4b5fb885c92e7a63ab702d5b6d38108d5cc9a7)